sqlalchemy_filters/models.py

```diff
@@ -61,20 +78,39 @@ def get_query_models(query):
         A dictionary with all the models included in the query.
     """
     models = [col_desc['entity'] for col_desc in query.column_descriptions]
-    models.extend(mapper.class_ for mapper in query._join_entities)
+
+    # account joined entities
+    if sqlalchemy_version_lt('1.4'):  # pragma: nocover
```

Comment:
It might be worth using this https://github.com/wemake-services/coverage-conditional-plugin instead of just `pragma: nocover`.

Reply:
Done